fsx-script/paket.dependencies

```diff
@@ -3,4 +3,12 @@ generate_load_scripts: true
 source https://www.nuget.org/api/v2
 framework: netstandard2.0
 
-nuget Fable.Elmish.React
\ No newline at end of file
+nuget Fable.Elmish.React
+
+// Needed for intellisense workaround in Ionide
+// See https://github.com/ionide/ionide-vscode-fsharp/issues/839#issuecomment-396296095
```

Comment:
You should be able to avoid this dependency.

In your `fsx` file you can just add something like:

Please note, that I took it from my FAKE script so you probably need to customize/remove the `#if !FAKE` condition

```fs
#if !FAKE 
#r "Facades/netstandard"
#r "netstandard"
#endif
```

I find it better that asking the user to reference a `.dll` user relative path which can changes depending on the project architecture.

Reply:
Tried that, `#r "netstandard"` doesn't work for me.
![image](https://user-images.githubusercontent.com/2621499/57767870-af8ad380-770a-11e9-9f6b-b2ee128bcb9d.png)

Reply:
Are you on OSX/Linux? If yes this is normal.

- `#r "Facades/netstandard"` is for supporting mono (OSX/Linux)
- `#r "netstandard"` is for supporting Windows

Because `mono` team put the facades, not in the same place.

Reply:
Hi, I'm on Windows. As mentioned `#r "netstandard"` doesn't work see screenshot.